extensions/wrapper/wrapper-common-api/src/main/java/de/sovity/edc/ext/wrapper/api/common/model/UiAssetCreateRequest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ *  Copyright (c) 2022 sovity GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       sovity GmbH - initial API and implementation
+ *
+ */
+
+package de.sovity.edc.ext.wrapper.api.common.model;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.Map;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Schema(description = "Type-Safe OpenAPI generator friendly Asset Create DTO that supports an opinionated\"\n" +
+        "        + \" subset of the original EDC Asset Entity.")
+public class UiAssetCreateRequest {
+    @Schema(description = "Data Address", requiredMode = Schema.RequiredMode.REQUIRED)
+    private Map<String, String> dataAddressProperties;
+
+    @Schema(description = "Properties of the Data Address", requiredMode = Schema.RequiredMode.REQUIRED)
+    private Map<String, String> properties;
+
+    @Schema(description = "Private Asset Properties", requiredMode = Schema.RequiredMode.REQUIRED)
+    private Map<String, String> privateProperties;
```

Comment:
should contain all important settable fields.

should not include the http proxy hints, they should be set automatically in the backend depending on the data address properties